src/strategies/masterchef/index.ts

```diff
@@ -0,0 +1,102 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+const MASTERCHEF_SUBGRAPH_URL = {
+  '1': 'https://api.thegraph.com/subgraphs/name/sushiswap/master-chef'
+};
+
+const SUSHISWAP_SUBGRAPH_URL = {
+  '1': 'https://api.thegraph.com/subgraphs/name/sushiswap/exchange'
+};
+
+export const author = '0xKiwi';
+export const version = '0.1.0';
+
+export async function strategy(
+  _space,
+  network,
+  _provider,
+  address,
```

Comment:
@0xKiwi This should not be `address` but `addresses` it's an array of voter addresses. The strategy must calculate score for each of these addresses not just a single one, the strategy should be refactored to reflect this now the code is made assuming this is just a single address.